Signal/src/ViewControllers/SettingsTableViewController.h

```diff
@@ -19,6 +19,7 @@
 @property (strong, nonatomic) IBOutlet UILabel *linkedDevicesLabel;
 @property (strong, nonatomic) IBOutlet UILabel *advancedLabel;
 @property (strong, nonatomic) IBOutlet UILabel *aboutLabel;
+@property (strong, nonatomic) IBOutlet UILabel *blocklistLabel;
```

Comment:
Done. There's some other places to fix this issue in SSK. I'll try to remember to do that.